docs/concepts/services/fetch.md

```diff
@@ -0,0 +1,236 @@
+---
+id: fetch
+title: The fetch service
+---
+## Introduction
+The fetch module can be used to make HTTP requests to a server. This enables applications to
+communicate with external services and persist data.
```

Comment:
Doesn't this sentence feel unfinished?

Reply:
@teymour-aldridge I never know if you simply disagree with the suggestion or if it's something else.
Even something as simple as :-1: would do the trick. If I feel strongly about the issue I will try to back it up with some arguments.
I would love to get this merged as soon as possible (even if the docs aren't even deployed to `yew.rs` right now).

Reply:
Sorry about that! No I agree with the suggestion – not sure why I didn't add it to the batch.

I'm not actually sure if the code example actually runs – I'd like to test it locally just to be sure :)

Reply:
That's great to hear, thank you. Please ping me when it's ready to merge.